### ISSUE TITLE
bug: Incorrect code completions on some methods.

### ISSUE THREAD

**Issue by shinkaihiroyuki** [kintone/js-sdk#245]
## Target Package

- @kintone/rest-api-client 1.3.0

## Environment

- Operating System: Windows 10 Pro / macOS Catalina
- Node.js version: v12.15.0
- Visual Studio Code v1.45.1

## Current Behavior

code completion on `record.updateAllRecords()` is not correct.

![2020-05-27_10h42_51](https://user-images.githubusercontent.com/16934699/82968567-9187cc80-a008-11ea-98f7-34df9df94751.png)

Some methods have the same issue, for example; `updateRecords()`, `updateRecordAssignees()`, `deleteRecords()`, `deleteAllRecords()`,  `deleteRecordComment()`, and so on.

## Expected Behavior

They should be `RecordID` and `RevisionID`.

**Comment by zaki-yama** [comment]
@shinkaihiroyuki 
Thank you for reporting.
There's a possibility that it's due to VSCode's bug, so we'll search if there're related issues in VSCode's repo.

**Comment by zaki-yama** [comment]
@shinkaihiroyuki 
We investigated and found that we could reproduce the problem with the following code that's unrelated to rest-api-client.

```ts
type Foo = string | number;
type Bar = string | number;

function someFunc(params: {
  foo: Foo,
  bar: Bar,
}) {
  //
}

// then hover here
someFunc(
```

![SS 2020-05-28 at 21 44 48](https://user-images.githubusercontent.com/1001444/83518203-20b74780-a515-11ea-9421-3964843d6a83.png)

So we created an issue in VSCode repository.
https://github.com/microsoft/vscode/issues/98727
(Now it was transferred to TypeScript's repo: https://github.com/microsoft/TypeScript/issues/38829)

We'll track it and close this issue.

Thank you for reporting.
